apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/ChatModel.kt

```diff
@@ -395,6 +394,26 @@ object ChatModel {
         addChat(chat)
       }
     }
+
+    fun addToChatItems(index: Int, elem: ChatItem) {
```

Comment:
these are moved inside Context class to avoid round trip in code figuring out current context